packages/api/start/routes/disclaimers.js

```diff
@@ -0,0 +1,428 @@
+/* eslint-disable jsdoc/check-tag-names */
+/* eslint-disable jsdoc/check-indentation */
+
+const { getMiddlewareRoles, roles } = require('../../app/Utils/roles_capabilities');
+
+const Route = use('Route');
+
+/** Disclaimer Routes */
+
+/**
+ * @api {get} /disclaimers/:id Gets a single Disclaimer
+ * @apiGroup Disclaimers
+ * @apiPermission ADMIN
+ * @apiHeader {String} Authorization Authorization Bearer Token.
+ * @apiHeaderExample {json} Header-Example:
+ *    {
+ *      "Authorization": "Bearer <token>"
+ *    }
+ * @apiParam {Number} id Mandatory Disclaimer ID.
+ * @apiParamExample  {json} Request sample:
+ *	/disclaimers/1
+ * @apiSuccess {Number} id Disclaimer ID
+ * @apiSuccess {Date} created_at Disclaimer Register date
+ * @apiSuccess {Date} updated_at Disclaimer Update date
+ * @apiSuccess {String} description Disclaimer Description
+ * @apiSuccess {Boolean} required Disclaimer Is required
+ * @apiSuccess {String="privacypolicy", "termsOfUseRegister", "termsOfUseTechnology"} type Disclaimer Type
+ * @apiSuccess {String} version Disclaimer Version
+ * @apiSuccessExample {json} Success
+ * HTTP/1.1 200 OK
+ *{
+ *    "id": 1,
+ *    "created_at": "2020-10-28 13:29:44",
+ *    "updated_at": "2020-10-28 13:36:45",
+ *    "description": "Declaro ciência dos Termos e Condições de Uso.",
+ *    "required": 1,
+ *    "type": "privacypolicy",
+ *    "version": "2"
+ *}
+ *@apiError (Bad Request 400) {Object} error Error object
+ *@apiError (Bad Request 400) {String} error.error_code Error code
+ *@apiError (Bad Request 400) {String} error.message Error message
+ *@apiErrorExample {json} Resource Disclaimer was not found
+ *    HTTP/1.1 400 Bad Request
+ *		{
+ * 			"error": {
+ *   			"error_code": "RESOURCE_NOT_FOUND",
+ *   			"message":"The resource Disclaimer was not found"
+ * 			}
+ *		}
+ */
+Route.get('disclaimers/:id', 'DisclaimerController.show').middleware(['handleParams']);
+/**
+ * @api {get} /disclaimers Lists All Disclaimers
+ * @apiGroup Disclaimers
+ * @apiPermission ADMIN
+ * @apiHeader {String} Authorization Authorization Bearer Token.
+ * @apiHeaderExample {json} Header-Example:
+ *    {
+ *      "Authorization": "Bearer <token>"
+ *    }
+ * @apiParam {Number} id Mandatory Disclaimer ID.
+ * @apiParamExample  {json} Request sample:
+ *	/disclaimers/1
```

Comment:
```suggestion
 * @apiParamExample  {json} Request sample:
 *	/disclaimers
```
Essa rota não é protegida